.github/workflows/reanimated-compatibility-check-nightly.yml

```diff
@@ -0,0 +1,73 @@
+name: Reanimated compatibility check [Nightly]
+env:
+  YARN_ENABLE_HARDENED_MODE: 0
+  YARN_ENABLE_IMMUTABLE_INSTALLS: 0
+on:
+  pull_request:
+    paths:
+      - .github/workflows/reanimated-compatibility-check-nightly.yml
+  schedule:
+    - cron: '37 19 * * *'
+  workflow_dispatch:
+
+jobs:
+  build_ios:
+    if: github.repository == 'software-mansion/react-native-reanimated'
+    runs-on: ${{matrix.platform == 'iOS' && 'macos-14' || 'ubuntu-latest'}}
+    strategy:
+      matrix:
+        platform: ['iOS', 'Android']
+        mode: ['Debug', 'Release']
+        # TODO: Fetch compatibility in docs and here from common sources.
+        # Update it when dropping support for RN versions.
+        react-native:
+          [
+            { version: '0.72', architecture: 'Paper' },
+            { version: '0.73', architecture: 'Paper' },
+            { version: '0.74', architecture: 'Paper' },
```

Comment:
Shouldn't we also check `fabric` with `0.74`?

Reply:
You are rite!